pkg/commonchecks/corechecks.go

```diff
@@ -90,7 +91,11 @@ func RegisterChecks(store workloadmeta.Component, tagger tagger.Component, cfg c
 	corecheckLoader.RegisterCheck(nvidia.CheckName, nvidia.Factory())
 	corecheckLoader.RegisterCheck(oracle.CheckName, oracle.Factory())
 	corecheckLoader.RegisterCheck(oracle.OracleDbmCheckName, oracle.Factory())
-	corecheckLoader.RegisterCheck(disk.CheckName, disk.Factory())
+	if cfg.GetBool("use_diskv2_check") {
+		corecheckLoader.RegisterCheck(disk.CheckName, diskv2.Factory())
+	} else {
+		corecheckLoader.RegisterCheck(disk.CheckName, disk.Factory())
+	}
```

Comment:
If we want to test both implementations at the same time to compare values, this approach would not allow us to do it. Should we look for a different approach that allows us to test both implementations at the same time?